src/Command/ReloadCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-swoole for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-swoole/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Zend\Expressive\Swoole\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Zend\Expressive\Swoole\SwooleRequestHandlerRunner;
+
+class ReloadCommand extends Command
+{
+    public const HELP = <<< 'EOH'
+Reload the web server. Sends a SIGUSR1 signal to all worker processes.
```

Comment:
@weierophinney Actually it is sends a SIGUSR1 signal to master process or manager process.